### DISCUSSION TITLE
Developing custom transformer

### DISCUSSION THREAD

**Discussion by TheBojda** [parcel-bundler/parcel#7210] [Q&A]
I tried to develop a custom transformer based on https://parceljs.org/features/plugins/ , but none of the methods (workspaces, link) worked for me.

Is there any working code sample available somewhere about how to do this?

If not, a git repo with a simple example (a simple app that uses a local transformer plugin that does nothing) would be very helpful.

**Comment by mischnic** [comment]
Here's an example for using workspaces: https://github.com/mischnic/parcel-transformer-lezer

**Reply by TheBojda** [reply]
Thank you @mischnic . 

I downloaded it. How can I start the example project?

**Reply by mischnic** [reply]
Run `yarn` in the root, go into `example`, run `yarn build`

**Reply by TheBojda** [reply]
Now your example is working fine for me (thank you for it), but parcel plugins are .js files that look like typescript (for example this one: https://github.com/parcel-bundler/parcel/blob/v2/packages/transformers/vue/src/VueTransformer.js). How can I use these kinds of transformers?

(I would like to modify the default VueTransformer and change the vue3 sfc compiler to the vue2 compiler.)
